WireDomain/Sources/WireDomain/Repositories/User/UserRepository.swift

```diff
@@ -89,6 +93,11 @@ public final class UserRepository: UserRepositoryProtocol {
         }
     }
 
+    public func removePushToken() {
+        // TODO: [WPB-10199] use new propertyWrapper for UserDefaults when related PR is merged
+        UserDefaults.standard.set(nil, forKey: "PushToken")
```

Comment:
can we take care of the TODO before merging?
also it would be better to use a constant for the key, in case we have to clean up it's easier to look for.

Reply:
yes this one is related to this PR [here](https://github.com/wireapp/wire-ios/pull/1932) that includes a **scoped** UserDefaults approach and a propertyWrapper. So keeping this one on hold for now

Reply:
done in https://github.com/wireapp/wire-ios/pull/1948/commits/09836061f1870849edc69be3d204a5c2afb69df8